src/Arcus.Security.Tests.Unit/KeyVault/KeyVaultSecretProviderTests.cs

```diff
@@ -25,38 +31,71 @@ public void KeyVaultSecretProvider_CreateWithHttpScheme_ShouldFailWithUriFormatE
             string uri = $"http://{Guid.NewGuid():N}.vault.azure.net/";
 
             // Act & Assert
-            Assert.ThrowsAny<UriFormatException>(() => new KeyVaultSecretProvider(null, new KeyVaultConfiguration(uri)));
+            Assert.ThrowsAny<UriFormatException>(() => new KeyVaultSecretProvider((IKeyVaultAuthentication) null, new KeyVaultConfiguration(uri)));
         }
 
         [Fact]
         public void KeyVaultSecretProvider_CreateWithoutUri_ShouldFailWithArgumentException()
         {
-            // Arrange
-            string uri = null;
-
             // Act & Assert
-            Assert.ThrowsAny<ArgumentException>(() => new KeyVaultSecretProvider(new AzureKeyVaultAuthenticatorDummy(), new KeyVaultConfiguration(uri)));
+            Assert.ThrowsAny<ArgumentException>(
```

Comment:
What's the added value of this?

Reply:
Formatting you mean? it helps when you place each separate argument on a single line so you don't confuse which argument is sent to which instance.

Reply:
Can you elaborate on that please? I'm not really sure I get the issue as intellisense would help when adding new parameters but I'm probably not getting the point

Reply:
No, it's not a point of intellesense or programmability; it's about readability. When all arguments are placed on the same line you can't tell in one instance which argument is sent to which instance because you first have to look where each paranthesis start and ends.
Placing them on a seperate line helps with seeing the structure in which an instance is build up.

Reply:
I think we don't need that but ok.